B827EBFFFE03284C.json

```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "eui-bcbcbcbcbcbcbcbc",
+    "servers": [
+      {
+        "server_address": "router.eu.thethings.network",
```

Comment:
If the gateway is in Switzerland, then you could use the Swiss router instead of the EU one, check the list here: https://www.thethingsnetwork.org/docs/gateways/packet-forwarder/semtech-udp.html#router-addresses